crates/proof-of-sql/src/sql/parse/query_context.rs

```diff
@@ -224,10 +224,6 @@ impl<C: Commitment> TryFrom<&QueryContext> for Option<GroupByExpr<C>> {
     type Error = ConversionError;
 
     fn try_from(value: &QueryContext) -> Result<Option<GroupByExpr<C>>, Self::Error> {
-        // Currently if there is no where clause, we can't prove the query
-        if value.where_expr.is_none() {
-            return Ok(None);
-        }
```

Comment:
I'm a bit confused by this removal, why was this code here in the first place - why are we removing it? Could you add an explanation to the PR description

Reply:
@tlovell-sxt Sure!